src/components/layout/navigation/navigation.tsx

```diff
@@ -241,7 +244,7 @@ const Navigation: React.FC<NavigationProps> = ({
             {t('navigation.imprint')}
           </LegalLink>
         </li>
-        <span>·</span>
+        <li>·</li>
```

Comment:
Think we should add `aria-hidden` here to avoid it being counted as a menu element.
<img width="90%" alt="3 elements" src="https://github.com/satellytes/satellytes.com/assets/1720843/9e247960-ce2b-4213-8b1d-d59a2dc0739a">